### PR TITLE
add github workflow

### PR DESCRIPTION
**Pull request by live627**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '5.4'
```
Comment (albertlast) [inline .github/workflows/php.yml:13]: Teste the less liky version is that a good idea?

File: other/check-smf-languages.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines https://www.simplemachines.org
+ * @copyright 2020 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 2.1 RC3
+ */
```
Comment (albertlast) [inline other/check-smf-languages.php:12]: I don't understand full was the intention of this test is,
maybe a description would help?

File: other/check-smf-license.php [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+ * @package SMF
+ * @author Simple Machines http://www.simplemachines.org
+ * @copyright 2020 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 2.1 RC3
+ */
+
+// Stuff we will ignore.
+$ignoreFiles = array(
```
Comment (albertlast) [inline other/check-smf-license.php:15]: Could we read the gitignore?

**Comment by live627** [comment]
Superseded by #6427

**Pull request closed without merging**